### ISSUE TITLE
[0.49.10] Unable to load save game

### ISSUE THREAD

**Issue by kipstafoo** [MegaMek/megamek#3916]
My MM game crashed after the physical phase and now it won't load. When I try to load it through MHQ, the MM instance will crash. If I try to load it load it simply by running MM, the bottom status screen says 'Waiting for physical attack phase' then does nothing else - no buttons active, nothing.

[The Dark Ones30520704.cpnx.gz](https://github.com/MegaMek/megamek/files/9677246/The.Dark.Ones30520704.cpnx.gz)
[autosave.sav.gz](https://github.com/MegaMek/megamek/files/9677248/autosave.sav.gz)

**Comment by HammerGS** [comment]
Any logs?

**Comment by kipstafoo** [comment]
> Any logs?

Dangit, I knew I forgot something. But, that being said, I went through the logs and there was nothing there other than standard debug logs. Sorry @HammerGS

**Comment by Windchild292** [comment]
We require logs to do anything for this.

**Comment by kipstafoo** [comment]
[legacy.log](https://github.com/MegaMek/megamek/files/9683549/legacy.log)
[megamek.log](https://github.com/MegaMek/megamek/files/9683550/megamek.log)
[mekhq.log](https://github.com/MegaMek/megamek/files/9683551/mekhq.log)
[megameklab.log](https://github.com/MegaMek/megamek/files/9683552/megameklab.log)
[gamelog.zip](https://github.com/MegaMek/megamek/files/9683554/gamelog.zip)

**Comment by Windchild292** [comment]
```
java.lang.NullPointerException
	at megamek.server.GameManager.preTreatPhysicalAttack(GameManager.java:31785)
	at megamek.server.GameManager.resolvePhysicalAttacks(GameManager.java:14440)
	at megamek.server.GameManager.endCurrentPhase(GameManager.java:2227)
	at megamek.server.GameManager.sendCurrentInfo(GameManager.java:696)
	at megamek.server.Server.sendCurrentInfo(Server.java:750)
	at megamek.server.Server.receivePlayerName(Server.java:710)
	at megamek.server.Server.handle(Server.java:1219)
	at megamek.server.Server$1.packetReceived(Server.java:224)
	at megamek.common.net.connections.AbstractConnection.processConnectionEvent(AbstractConnection.java:550)
	at megamek.common.net.connections.AbstractConnection.processPacket(AbstractConnection.java:458)
	at megamek.common.net.connections.AbstractConnection.update(AbstractConnection.java:405)
	at megamek.server.ConnectionHandler.run(ConnectionHandler.java:47)
	at java.base/java.lang.Thread.run(Thread.java:829)
s: error reading command: null
```

**Comment by Windchild292** [comment]
Resolved by #3964
